contracts/Whitelist.sol

```diff
@@ -167,7 +171,7 @@ contract Whitelist is Ownable {
 
     /**
      * @notice allows the owner to whitelist a collateral address
-     * @dev can only be called from the owner address
+     * @dev can only be called from the owner address. Don't whitelist otokens as collateral in the current version.
```

Comment:
Why do we want to add this description? otokens are never meant to be a collateral in our system right?

Reply:
Yeah I think better to update this comment. Otoken are already collateral in our system, but they are used in the long Otoken array, so maybe make sense to just explain that this functions is meant for collateral assets that are not an Otoken?